src/test/java/com/intuit/graphql/orchestrator/federation/EntityDataFetcherTest.java

```diff
@@ -181,6 +184,131 @@ public void get_no_entity_field_exception() throws ExecutionException, Interrupt
     }
   }
 
+  @Test
+  public void get_null_entity_field_exception() throws ExecutionException, InterruptedException {
```

Comment:
Might want to add  integration test where EntityDataFetcher get an `error` and empty data.

Reply:
I'll fix that in my next commit